modules/menu/menu.c

```diff
@@ -602,6 +637,7 @@ static const struct cmd callcmdv[] = {
 	{'r', CMD_IPRM,"Transfer call",       call_xfer             },
 	{'x',       0, "Call hold",           call_holdresume       },
 	{'H',       0, "Hold previous call",  hold_prev_call        },
+	{'A', CMD_IPRM,"Switch audio device", switch_audio_dev      },
```

Comment:
I think we have run out of letters now :)

the auloop module is using 'a' and 'A'...

as Doug has pointed out we should now add support for an additional
set of commands with long strings, in addition to the current one.